adapters/oidc/spring-security/src/main/java/org/keycloak/adapters/springsecurity/filter/KeycloakAuthenticationProcessingFilter.java

```diff
@@ -207,28 +216,17 @@ protected void successfulAuthentication(HttpServletRequest request, HttpServletR
         } finally {
             SecurityContextHolder.clearContext();
         }
-
     }
+    
+    
 
     @Override
-    protected void unsuccessfulAuthentication(HttpServletRequest request, HttpServletResponse response,
-            AuthenticationException failed) throws IOException, ServletException {
-
-        if (this.isBearerTokenRequest(request)) {
-            SecurityContextHolder.clearContext();
-            response.sendError(HttpServletResponse.SC_UNAUTHORIZED, "Unable to authenticate bearer token");
-            return;
-        }
-        else if (this.isBasicAuthRequest(request)) {
-            SecurityContextHolder.clearContext();
-            response.sendError(HttpServletResponse.SC_UNAUTHORIZED, "Unable to authenticate with basic authentication");
-            return;
-        }
-
-        super.unsuccessfulAuthentication(request, response, failed);
-    }
+	protected void unsuccessfulAuthentication(HttpServletRequest request, HttpServletResponse response,
+			AuthenticationException failed) throws IOException, ServletException {
+		super.unsuccessfulAuthentication(request, response, failed);
+	}
 
-    @Override
+	@Override
```

Comment:
Yep, I changed my IDE conf to apply the "JBoss Community" style. However, formatting the file automatically cause many changes. Do you want me to commit these changes for this PR regarding to your git recommandations ?
> - One commit per PR
> - No changes to code not directly related to your change (e.g. no formatting changes or refactoring to existing code, if you want to refactor/improve existing code that's a separate discussion to mailing list and JIRA issue)